engine/crates/telemetry/src/metrics/request.rs

```diff
@@ -17,12 +19,17 @@ pub struct RequestMetricsAttributes {
     pub cache_status: Option<String>,
     pub gql_status: Option<GraphqlResponseStatus>,
     pub client: Option<Client>,
+    pub url_scheme: Option<String>,
+    pub route: Option<String>,
```

Comment:
Think could be a `&'static str`. Not sure we'll ever need a dynamic string even in the future

Reply:
Ummm... we do. https://grafbase.com/docs/self-hosted-gateway#graph-configuration

Reply:
hum... ok fair point forgot that was configurable. Could `Box::leak` I guess but with config reloading and all...